src/dashboard/src/components/forms/select/Select.tsx

```diff
@@ -6,7 +6,7 @@ import { FormError, IOption } from '..';
 import { generateKey } from './utils';
 
 export interface ISelectProps<T> {
-  simple?: boolean; 
+  variant?: 'primary' | 'filter';
```

Comment:
Changed from boolean to a variant to remain consistent with other implementations.